feature-libs/order/components/order-details/myaccount-v2/download-invoices/myaccount-v2-download-invoices.module.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { inject, NgModule } from '@angular/core';
+import { I18nModule, provideDefaultConfig } from '@spartacus/core';
+import { PDFInvoicesComponentsModule } from '@spartacus/pdf-invoices/components';
+import {
+  IconModule,
+  KeyboardFocusModule,
+  PaginationModule,
+  SortingModule,
+  SpinnerModule,
+} from '@spartacus/storefront';
+import { defaultMyAccountV2DownloadInvoicesLayoutConfig } from './default-myaccount-v2-download-invoices-layout.config';
+import { MyAccountV2DownloadInvoicesEventListener } from './myaccount-v2-download-invoices-event.listener';
+import { MyAccountV2DownloadInvoicesComponent } from './myaccount-v2-download-invoices.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    KeyboardFocusModule,
+    IconModule,
+    I18nModule,
+    PaginationModule,
+    SortingModule,
+    SpinnerModule,
+    PDFInvoicesComponentsModule,
+  ],
+  providers: [
+    provideDefaultConfig(defaultMyAccountV2DownloadInvoicesLayoutConfig),
+  ],
+  exports: [MyAccountV2DownloadInvoicesComponent],
+  declarations: [MyAccountV2DownloadInvoicesComponent],
+})
+export class DownloadOrderInvoicesDialogModule {
```

Comment:
[MEDIUM]
Let's align the filename with the class name:
- "download invoices" vs "download order invoices"
- "my account v2" vs _no_ "my account v2"

Reply:
done